packages/storage/amplify_storage_s3/example/integration_test/main_test.dart

```diff
@@ -498,16 +506,20 @@ void main() async {
             );
           });
 
-          testWidgets('get url of object with access level guest',
-              (WidgetTester tester) async {
+          testWidgets(
```

Comment:
To Travis's earlier point, why are we replacing existing tests instead of adding new ones? It seems like you've flattened tests into guest+no acceleration / authorized+acceleration. Is there any harm making this 2 different dimensions so we get a full matrix of acceleration/authorization cases?

Reply:
Yea, can just add a separate set upload-download-delete test case on accelerate endpoint for clarity, I changed existing tests as acceleration doesn't involve or is not involved with auth.

Reply:
That's fine I would just add it as a separate test, then, so:
- guest
- authorized
- accelerate